Dockerfile

```diff
@@ -19,10 +16,14 @@ RUN  go mod download
 # Copy the go source
 COPY . $work_dir/
 # Build
-RUN  go build -a -o $work_dir/bin/controller $work_dir/services/$service_alias/cmd/controller/main.go
+RUN GIT_VERSION=$(git describe --tags --dirty --always) && \
+    GIT_COMMIT=$(git rev-parse HEAD) && \
+    BUILD_DATE=$(date +%Y-%m-%dT%H:%M:%S%z) && \
+    CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build \
+    -ldflags="-X ${VERSION_PKG}.GitVersion=${GIT_VERSION} -X ${VERSION_PKG}.GitCommit=${GIT_COMMIT} -X ${VERSION_PKG}.BuildDate=${BUILD_DATE}" -a -o $work_dir/bin/controller $work_dir/services/$service_alias/cmd/controller/main.go
```

Comment:
nit: i'm fine with this change but i find it easier to read when the environment variables are declared separately

Reply:
I'm actually not fine with this change... @nithu0115 is there a reason you changed this like so instead of adding another environment variable?